app/main.py

```diff
@@ -1,13 +1,14 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return dict(line=error["line_number"], column=error["column_number"],
```

Comment:
```suggestion
    return {
         "line": error["line_number"], 
         "column": error["column_number"],
         ...
     }
```
it's preferable to use literal syntax `{}` to create a dictionary because it works faster